low-level/solution/lr2/server/manifest.py

```diff
@@ -197,6 +199,8 @@ def get_server_hw_info(self):
         # Adding data to kv
         output_file.set(self.kv_dict.keys(), self.kv_dict.values())
         lshw_data = self.get_manifest_output_data()
+        if "disk" in lshw_data["hw"]:
```

Comment:
add comment here, stating here storage enclosure (RBOD/JBOD) drives are filtered out, as those needs to be represented separately & uniquely

Reply:
Done